src/aiokatcp/server.py

```diff
@@ -64,6 +64,8 @@
 class ClientConnection(connection.Connection):
     """Server's view of the connection from a single client."""
 
+    owner: "DeviceServer"
```

Comment:
To get a bit pedantic/academic, your reasoning for making this a class attribute rather than an instance attribute?

Reply:
It is an instance attribute. If it was a class attribute, it would be declared
```
    owner: ClassVar["DeviceServer"]
```
I added this to refine the type information: from the base class, typing tools would only know that it is a `_ConnectionOwner[ClientConnection]`.

Reply:
Interesting.